### ISSUE TITLE
[Feature Request]: Pass ALT text value as title automatically for inline images

### ISSUE THREAD

**Issue by florent-leborgne** [elastic/docs-builder#1655]
### Prerequisites

- [x] I have searched existing issues to ensure this feature hasn't already been requested
- [x] I have tested using the latest version of docs-builder

### What problem are you trying to solve?

Is there a way we can automatically pass image ALT values as title values as well? Example:
`![Hide missing values](../images/charts-gaps-fill-hide.png "title =50%")` shows this when hovering an image:

<img width="720" height="427" alt="Image" src="https://github.com/user-attachments/assets/ab4b22a1-fafa-45fd-b253-672a25682aee" />

`<img src="/elastic/docs-content/pull/2210/explore-analyze/images/charts-gaps-fill-hide.png" width="50%" height="50%" alt="Hide missing values" title="title">`

- We must resize our images in many cases, so if we [start from the example in the internal docs](https://elastic.github.io/docs-builder/syntax/images/#inline-image-sizing), we all use `"title =<the-size>"` so we'll see a proliferation of "title" titles.

Alternatively, if we know we don't want to do this or cannot, let us know as well so we can plan to check these "title" values in existing docs and improve our accessibility/SEO

### Proposed Solution

- Automatically pass the ALT value as title if possible
- Update the internal docs: the "inline image sizing" should focus on the size parameter and exclude the dummy title, if that is even possible to specify the size without a title

### How important is this feature to you?

Nice to have
